snapcraft/cmds.py

```diff
@@ -229,7 +229,7 @@ def run(args):
             ['ssh'] + ssh_opts +
             ['ubuntu@localhost', '-p', '8022', 'sudo snappy install  *.snap'])
         # "login"
-        _check_call(
+        _call(
```

Comment:
I was trying to see what exception was thrown as I wasn't seeing this anymore. I wanted to fine grain the exception cause if possible.